docs/spec_draft.md

```diff
@@ -682,6 +721,43 @@ specification. Numeric precision is implementation-defined.
 
 [Back to Ops](#index-of-documented-ops)
 
+## stablehlo.sqrt
+
+`stablehlo.sqrt(operand) -> result`
+
+### Semantics
+
+Performs element-wise square root operation on `operand` tensor and produces a
+`result` tensor, implementing the `sqrt` operation from the IEEE-754
```

Comment:
I could not find  `sqrt` as a standalone operation in the standard. Once it is used in the formula of `rsqrt`. 
However, I can find the name of the operation as `squareRoot`.

Reply:
Fixed